docs/partial_source/deep_dive/16_ivy_frontends.rst

```diff
@@ -420,6 +417,11 @@ not necessary to uniquely flag every single NumPy function as supporting only CP
 as this is a limitation of the entire framework, and this limitation is already
 `globally flagged <https://github.com/unifyai/ivy/blob/6eb2cadf04f06aace9118804100b0928dc71320c/ivy/functional/backends/numpy/__init__.py#L21>`_.
 
+Another common case, If the function supports a data type, but a backend framework does not. the frontend function `jax.lax.cumprod <https://github.com/unifyai/ivy/blob/6e80b20d27d26b67a3876735c3e4cd9a1d38a0e9/ivy/functional/frontends/jax/lax/operators.py#L111>`_
```

Comment:
Double check the grammar here. You have a capital letter after a comma, and then a non-capital letter after a comma. In general it is quite hard to follow along with what the overall sentence is saying. Please try and restructure it for maximal clarity :)